test/e2e/helpers.js

```diff
@@ -861,20 +861,20 @@ const TEST_SEED_PHRASE_TWO =
 
 // Usually happens when onboarded to make sure the state is retrieved from metamaskState properly, or after txn is made
 const locateAccountBalanceDOM = async (driver, ganacheServer) => {
-  const balance = await ganacheServer.getBalance();
-
-  await driver.waitForSelector({
-    css: '[data-testid="eth-overview__primary-currency"]',
-    text: `${balance} ETH`,
-  });
+  const balanceSelector = '[data-testid="eth-overview__primary-currency"]';
+  if (ganacheServer) {
```

Comment:
There are situations there the account is not having the balance allocated. eg: we create 2nd account. So to re-use the selector logic, I combined this with `waitForAccountRendered` downstairs